### PR TITLE
Add a command line for api-documenter

### PR DESCRIPTION
**Pull request by pgonzal**
We need an easier way for people to invoke the **api-documenter** tool, that does not require enlisting and building the product code.

This PR also moves/renames some source files to make the project easier to understand.

### CODE REVIEW

File: libraries/api-documenter/ad.cmd [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@ECHO OFF
```
Comment (qz2017) [inline libraries/api-documenter/ad.cmd]: The name "ad" is kind of too brief.

**Comment by iclanton** [comment]
> 

Add the real binary and make sure it's not gitignored.

---
Refers to: libraries/api-documenter/ad.cmd:4 in 7a13657. [](commit_id = 7a1365775bca424fbec28c395c6157e9f6eb944d, deletion_comment = False)

File: libraries/api-documenter/ad.cmd (hunk 1 above)
Comment (pgonzal) [reply libraries/api-documenter/ad.cmd]: The real executable is "api-documenter".  This ad.cmd file is just used by me to invoke the tool locally when debugging it.

---
In reply to: [145812638](https://github.com/Microsoft/web-build-tools/pull/399#discussion_r145812638) [](ancestors = 145812638)

**Comment by pgonzal** [comment]
:shipit:

**Pull request merged**